src/components/inputs/YearsCounter/YearsCounter.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+
+import './styles.css';
+
+function YearsCounter({ setCount, count }: { setCount: any; count: number }) {
```

Comment:
```suggestion
function YearsCounter({ setCount, count }: { setCount: () => void; count: number }) {
```

Reply:
this isnt actually working for the useState functions... it recommends something like Dispatch<SetState<inffered type from default state>>